From matthias@kaehlcke.net  Tue Apr 21 22:57:08 2009
From: Matthias Kaehlcke <matthias@kaehlcke.net>
Date: Wed, 15 Apr 2009 22:28:36 +0200
Subject: USB: ci13xxx_udc: fix build error
To: Greg KH <greg@kroah.com>, dbrownell@users.sourceforge.net
Message-ID: <20090415202836.GV8693@traven>
Content-Disposition: inline


Fix build error in the MIPS USB IP core family device controller driver.
The driver calls udelay() without including linux/delay.h

Signed-off-by: Matthias Kaehlcke <matthias@kaehlcke.net>
Acked-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/ci13xxx_udc.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/gadget/ci13xxx_udc.c
+++ b/drivers/usb/gadget/ci13xxx_udc.c
@@ -51,6 +51,7 @@
  * - Gadget API (majority of optional features)
  * - Suspend & Remote Wakeup
  */
+#include <linux/delay.h>
 #include <linux/device.h>
 #include <linux/dmapool.h>
 #include <linux/dma-mapping.h>
